### PR TITLE
#113: add verification of minimalScore

### PR DESCRIPTION
**Pull request by rombautt**

### CODE REVIEW

File: foundations/src/test/java/io/yields/math/framework/ExplorerTest.java [hunk 1]
```diff
@@ -223,7 +223,7 @@ public void checkRangeDuringSampling() {
         PropertySupplier<Double> propertySupplier = doubleFunctionalContext -> singletonList(property);
 
         return new Explorer<>("name", "group", false, propertySupplier, nrOfSims, runsPerBatch, dataProvider, new SingleThreadedPropertyVerifier(), score,
-                              null);
+                              null, 1d);
```
Comment (jgheerardyn) [inline foundations/src/test/java/io/yields/math/framework/ExplorerTest.java]: Could you add a test where the minimal score is not reached and verify that the unit test actually fails?

**Comment by jgheerardyn** [comment]
reviewed

File: foundations/src/test/java/io/yields/math/framework/ExplorerTest.java (hunk 1 above)
Comment (rombautt) [inline foundations/src/test/java/io/yields/math/framework/ExplorerTest.java]: ofcourse, added

**Comment by rombautt** [comment]
@jgheerardyn updated wiki, please let me know if this is clear: https://github.com/jgheerardyn/yields/wiki/TestDefinitionRunner

**Comment by jgheerardyn** [comment]
@rombautt ok - it is clear how to do it when you have the test definition json file located on the Jenkins server. I understand that pushing such file to the Jenkins server from testpad and subsequently running the tests is not yet supported. Is this correct?

**Comment by rombautt** [comment]
Testpad would kick the build, not with a file as argument, but straight with the json content, that's why the runner supports both. No, it's not yet possible to start this build from testpad as it first needs to implement the screens for the testdefinitions.

**Comment by jgheerardyn** [comment]
ok clear thanks!

**Pull request merged**